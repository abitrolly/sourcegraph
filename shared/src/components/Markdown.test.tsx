import React from 'react'
import { Markdown } from './Markdown'
import { createMemoryHistory } from 'history'
import { mount } from 'enzyme'

describe('Markdown', () => {
    it('renders', () => {
        const history = createMemoryHistory()
        expect(mount(<Markdown history={history} dangerousInnerHTML="hello" />)).toMatchSnapshot()
    })
})
