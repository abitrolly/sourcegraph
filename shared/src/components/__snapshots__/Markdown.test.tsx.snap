// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown renders 1`] = `
<Markdown
  dangerousInnerHTML="hello"
  history="[History]"
>
  <div
    className="markdown"
    dangerouslySetInnerHTML={
      Object {
        "__html": "hello",
      }
    }
    onClick={[Function]}
  />
</Markdown>
`;
